resolwe_bio/tools/species_to_gsize.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python2
+# pylint: disable=missing-docstring,invalid-name
+# XXX: Refactor to a comand line tool and remove pylint disable
+"""Get genome size from species."""
+from __future__ import absolute_import, division, print_function
+
+import argparse
+
+parser = argparse.ArgumentParser(description="Get genome size from species")
+parser.add_argument('species', help="Species")
+args = parser.parse_args()
+
+
+def gsize_organism(name):
+    """Return genome size of species."""
+    mapping = {
+        'homo sapiens': '2.7e9',
+        'mus musculus': '1.87e9',
+        'dictyostelium discoideum': '3.4e7',
+        'drosophila melanogaster': '1.2e8',
+        'caenorhabditis elegans': '9e7',
+        'rattus norvegicus': '2e9',
+    }
+    return mapping[name.lower()]
+
+
+print(gsize_organism(args.species))
```

Comment:
When this will get `species` value that is not present in the `mapping` it will fail with KeyError. It would be nice if you could raise a more descriptive error.